experimental/framework/frs-client/src/frsClient.ts

```diff
@@ -0,0 +1,157 @@
+        containerSchema: ContainerSchema,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const container = await this.getContainerCore(
+            serviceContainerConfig,
+            runtimeFactory,
+            false,
+        );
+
+        const rootDataObject = await this.getRootDataObject(container);
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    private async getRootDataObject(
+        container: Container,
+    ): Promise<RootDataObject> {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return rootDataObject as RootDataObject;
+    }
+
+    private async getContainerCore(
+        routerliciousContainerConfig: FRSContainerConfig,
+        containerRuntimeFactory: IRuntimeFactory,
+        createNew: boolean,
+    ): Promise<Container> {
+        const module = { fluidExport: containerRuntimeFactory };
+        const codeLoader = { load: async () => module };
+
+        const loader = new Loader({
+            urlResolver: this.serviceConnectionConfig.urlResolver,
+            documentServiceFactory: this.serviceConnectionConfig.documentServiceFactory,
+            codeLoader,
+            logger: routerliciousContainerConfig.logger,
+        });
+
+        let container: Container;
+
+        if (createNew) {
+            // We're not actually using the code proposal (our code loader always loads the same module
+            // regardless of the proposal), but the Container will only give us a NullRuntime if there's
+            // no proposal.  So we'll use a fake proposal.
+            container = await loader.createDetachedContainer({
+                package: "no-dynamic-package",
+                config: {},
+            });
+            await container.attach({ url: routerliciousContainerConfig.id });
+        } else {
+            // Request must be appropriate and parseable by resolver.
+            container = await loader.resolve({ url: routerliciousContainerConfig.id });
+            // If we didn't create the container properly, then it won't function correctly.  So we'll throw if we got a
+            // new container here, where we expect this to be loading an existing container.
+            if (container.existing === undefined) {
+                throw new Error("Attempted to load a non-existing container");
+            }
+        }
+        return container;
+    }
+}
+
+/**
+ * FRSClient static class with singular global instance that lets the developer define
+ * all Container interactions with the FRS service
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class FRSClient {
+    private static globalInstance: FRSClientInstance | undefined;
+
+    static init(serviceConnectionConfig: RouterliciousService) {
+        if (FRSClient.globalInstance) {
+            console.log(
```

Comment:
Please no console logs. If you really need debug logging, please use debug library.
But even better - please use telemetry logger

Reply:
This code is still VERY early. I should have done this as a draft PR.